woocommerce/woocommerce
# Fixed notice when variation does not exists on frontend

### All Submissions:






### Changes proposed in this Pull Request:



Closes #20752.

### How to test the changes in this Pull Request:

1. Create a product variable with 50 variations.
2. Try select one combination that does not exist.
3. Check that there is no message, and it's impossible to add to the cart.
4. Switch to this branch and select the same non-existing combination again, now message are displayed.

### Other information:




### Changelog entry

> Fixed missing notice about no product variation matching
